deepeval/cli/test.py

```diff
@@ -49,16 +49,22 @@ def run(
         "-n",
         help="Number of processes to use with pytest",
     ),
+    repeat: int = typer.Option(
```

Comment:
I updated this file for the new flag, i think the easiest for conflicts right now would just be to copy and paste the new file entirely then add the remaining logic for caching.